test/new-e2e/tests/cws/e2e_cws_test.go

```diff
@@ -97,10 +97,7 @@ func (a *agentSuite) TestOpenSignal() {
 	require.NoError(a.T(), err, "Signal rule creation failed")
 	a.signalRuleID = res2.GetId()
 
-	// Check if the agent has started
-	err = a.Env().Agent.WaitForReadyTimeout(1 * time.Minute)
-	require.NoError(a.T(), err)
-
+	// Check if the agent has
```

Comment:
Note for myself: remove this leftover
```suggestion
	// Check if the agent is ready
```